app/static/styles/app.css

```diff
@@ -4,6 +4,11 @@ nav {
 }
 
 @media -sass-debug-info{filename{}line{font-family:\0000318}}
+.body.container {
```

Comment:
No don't do this. Some libraries we depend on use css files. Also, not everyone who clones our code should need a sass compiler installed to run the app.